### ISSUE TITLE
Navigation and linkability

### ISSUE THREAD

**Issue by binford2k** [csb324/holyfucktheelectionistomorrow#34]
This should update browser history as the pages transition so that the back and forward buttons work as expected. It should also update the location hash to make subsections hot linkable.

I'd like to be able to share some actions with others without having to say "ok, first click on fuck shit up, then click on the earth is on fire, then click time....."

**Comment by oliverturner** [comment]
I like this idea... @csb324 would a PR help?

**Comment by jgstew** [comment]
I agree that the ability to have a deep link into a particular section would be useful.

**Comment by oliverturner** [comment]
ok folks... I'll try to find the time to implement this.

**Comment by oliverturner** [comment]
...aaaaand done! 

PR: #43 
Demo: https://hfte.codedsignal.co.uk
